src/main/windows/windowManager.ts

```diff
@@ -437,12 +452,52 @@ export function handleHistory(event: IpcMainEvent, offset: number) {
                 activeView.view.webContents.goToOffset(offset);
             } catch (error) {
                 log.error(error);
-                activeView.load(activeView.server.url);
+                activeView.load(activeView.tab.url);
             }
         }
     }
 }
+
+export function selectNextTab() {
+    const currentView = status.viewManager?.getCurrentView();
+    if (!currentView) {
+        return;
+    }
+
+    const currentTeamTabs = status.config?.teams.find((team) => team.name === currentView.tab.server.name)?.tabs;
+    const currentTab = currentTeamTabs?.find((tab) => tab.name === currentView.tab.type);
+    if (!currentTeamTabs || !currentTab) {
+        return;
+    }
+
+    const currentOrder = currentTab.order;
+    const nextOrder = ((currentOrder + 1) % currentTeamTabs.length);
+    const nextIndex = currentTeamTabs.findIndex((tab) => tab.order === nextOrder);
+    const newTab = currentTeamTabs[nextIndex];
+    switchTab(currentView.tab.server.name, newTab.name);
+}
+
+export function selectPreviousTab() {
+    const currentView = status.viewManager?.getCurrentView();
+    if (!currentView) {
+        return;
+    }
+
+    const currentTeamTabs = status.config?.teams.find((team) => team.name === currentView.tab.server.name)?.tabs;
+    const currentTab = currentTeamTabs?.find((tab) => tab.name === currentView.tab.type);
+    if (!currentTeamTabs || !currentTab) {
+        return;
+    }
+
+    const currentOrder = currentTab.order;
+
+    // js modulo operator returns a negative number if result is negative, so we have to ensure it's positive
+    const nextOrder = ((currentTeamTabs.length + (currentOrder - 1)) % currentTeamTabs.length);
```

Comment:
isn't this assuming that all tabs are open? although maybe you were planning on adding support for opening/closing tabs in a different PR

Reply:
Yes, that's a different PR